torchao/_models/llama/tokenizer.py

```diff
@@ -1,14 +1,8 @@
 # copied from https://github.com/pytorch-labs/gpt-fast/blob/main/tokenizer.py
+from __future__ import annotations
```

Comment:
which python version are you using?

Reply:
I'm using 3.9. I saw that that was what you are using in CI.

https://github.com/pytorch/ao/blob/e2301e9dba91fa962d673fdc3b3f0002856a3ba7/.github/workflows/ruff_linter.yml#L18

I think you are right that for what I did, this wouldn't be required for 3.9. I will remove it.

```suggestion
```